### PR TITLE
Add mne-qt-browser

### PR DESCRIPTION
**Pull request by larsoner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by larsoner** [comment]
@marsipu you need to comment here that you're willing to maintain this conda-forge recipe

File: recipes/mne-qt-browser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - mne_qt_browser = mne_qt_browser.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - mne-base
```
Comment (larsoner) [inline recipes/mne-qt-browser/meta.yaml:26]: @marsipu once we release 0.24 we should remember to update this!

File: recipes/mne-qt-browser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% set name = "mne-qt-browser" %}
+{% set version = "0.1.1" %}
```
Comment (marsipu) [inline recipes/mne-qt-browser/meta.yaml:2]: I updated the package once more.
```suggestion
{% set version = "0.1.2" %}
```

**Comment by marsipu** [comment]
Thanks a lot @larsoner for setting this up.

**Comment by marsipu** [comment]
I am willing to maintain this conda-forge recipe.

File: recipes/mne-qt-browser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{% set name = "mne-qt-browser" %}
+{% set version = "0.1.3" %}
```
Comment (marsipu) [inline recipes/mne-qt-browser/meta.yaml:2]:
```suggestion
{% set version = "0.1.4" %}
```

I increased the version-number once more yesterday for the newest feedback on the now merged PR in MNE-Python.

**Review by marsipu — CHANGES_REQUESTED** [review]

**Comment by larsoner** [comment]
@conda-forge/staged-recipes, this one seems to be ready for review. Thanks!

File: recipes/mne-qt-browser/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  noarch: python
+  entry_points:
+    - "mne-qt-browser = mne_qt_browser.__main__:main"
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (ocefpaf) [inline recipes/mne-qt-browser/meta.yaml:22]: Please add `>=3.7` here too.

**Comment by larsoner** [comment]
Done, thanks for the review @ocefpaf

**Pull request merged**